### PR TITLE
Added init script for database models

### PR DESCRIPTION
**Pull request by Naapperas**
This PR adds a script that takes a MongoDB dump and populates the local database with that data.

### CODE REVIEW

File: seed.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/env bash
```
Comment (dsantosferreira) [inline seed.sh]: Had to replace the shebang to `#!/usr/bin/env bash` in macOS. Don't know if that should be considered an issue or not

**Review by dsantosferreira — DISMISSED** [review]
Looks good to me! Left you with one small question

File: seed.sh (hunk 1 above)
Comment (Naapperas) [reply seed.sh]: MacOS moment, will change that.

**Review by dsantosferreira — APPROVED** [review]
Looks good to me!

**Pull request merged**